Src/MakingSense.SafeBrowsing/MakingSense.SafeBrowsing.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(LibraryFrameworks)'==''">net45;net40;net35;netstandard1.0;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks Condition="'$(LibraryFrameworks)'==''">net45;netstandard1.3</TargetFrameworks>
```

Comment:
I have said you to remove support for older frameworks. But, then I have remembered that we want to use this library in Doppler (I do not remember what .NET version it is being used there).

I know that the new dependency is not supported by older frameworks, but maybe in the future, we could add a workaround for them. 

So, what do you think about keeping support for older frameworks but do not include the new classes in that packages? 

I can easily create a commit restoring old framework support if you prefer.